[2019-12-30 15:34:33,154] [log.py|__console] [line:40] INFO    : 当前执行测试用例ID-> test_login001.1 ; 测试点-> 手机号或密码不匹配
[2019-12-30 15:34:34,188] [log.py|__console] [line:40] INFO    : 检查点-> 您输入的用户名或密码有误。
[2019-12-30 15:34:34,267] [log.py|__console] [line:40] INFO    : 异常登录，返回实际结果是->: 您输入的用户名或密码有误。
